examples/dt/uni01alpha/control-plane/service-values.yaml

```diff
@@ -0,0 +1,140 @@
+  name: service-values
+  annotations:
+    config.kubernetes.io/local-config: "true"
+
+data:
+  cinderVolumes:
+    lvm-iscsi:
+      replicas: 1
+      nodeSelector:
+        openstack.org/cinder-lvm: ""
+      customServiceConfig: |
+        [lvm]
+        image_volume_cache_enabled = false
+        volume_driver = cinder.volume.drivers.lvm.LVMVolumeDriver
+        volume_group = cinder-volumes
+        target_protocol = iscsi
+        target_helper = lioadm
+        volume_backend_name = lvm_iscsi
+        target_ip_address=172.18.0.5
+        target_secondary_ip_addresses = 172.19.0.5
+
+  cinderBackup:
+    customServiceConfig: |
+      [DEFAULT]
+      backup_driver = cinder.backup.drivers.swift.SwiftBackupDriver
+    replicas: 1
+
+  glance:
+    customServiceConfig: |
+      [DEFAULT]
+      enabled_backends = default_backend:swift
+
+      [glance_store]
+      default_backend = default_backend
+
+      [default_backend]
+      swift_store_create_container_on_put = True
+      swift_store_auth_version = 3
+      swift_store_auth_address = {{ .KeystoneInternalURL }}
+      swift_store_endpoint_type = internalURL
+      swift_store_user = service:glance
+      swift_store_key = {{ .ServicePassword }}
+    default:
+      replicas: 1
+
+  ironic:
+    enabled: true
+
+  swift:
+    enabled: true
+
+  octavia:
+    enabled: true
+
+  heat:
+    enabled: true
+
+  telemetry:
+    enabled: true
+    metricStorage:
+      enabled: true
+    autoscaling:
+      enabled: false
+    ceilometer:
+      enabled: true
+
+  ovn:
+    ovnController:
+      availability-zones:
+        - zone-1
+
+  neutron:
+    customServiceConfig: |
+      [DEFAULT]
+      dns_domain = openstackgate.local
+      vlan_transparent = true
+      service_plugins = qos,ovn-router,trunk,segments,port_forwarding,log
+      dhcp_agent_notification = false
+      agent_down_time = 600
+      router_distributed = true
+      router_scheduler_driver = neutron.scheduler.l3_agent_scheduler.ChanceScheduler
+      allow_automatic_l3agent_failover = true
+      debug = true
+      default_availability_zones = zone-1,zone-2
+      global_physnet_mtu = 9000
+
+      [agent]
+      report_interval = 300
+
+      [database]
+      max_retries = -1
+      db_max_retries = -1
+
+      [keystone_authtoken]
+      region_name = regionOne
+      memcache_use_advanced_pool = True
+      auth_type = password
+
+      [nova]
+      region_name = regionOne
+      endpoint_type = internal
+
+      [oslo_messaging_notifications]
+      driver = noop
+
+      [oslo_middleware]
+      enable_proxy_headers_parsing = true
+
+      [oslo_policy]
+      policy_file = /etc/neutron/policy.yaml
+
+      [placement]
+      region_name = regionOne
+
+      [ovs]
+      igmp_snooping_enable = true
+
+      [ovn]
+      ovsdb_probe_interval = 60000
+      ovn_emit_need_to_frag = true
+      ovn_metadata_enabled = true
+      enable_distributed_floating_ip = true
+
+      [ml2]
+      path_mtu = 9000
```

Comment:
as we agreed on the meeting, we can use the default mtu and omit this path_mtu = 9000 setting. For neutron mtu tests we can use default mtu from default network definition, just need to create public network for the job with a value smaller than geneve tenant networks.